### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by Blake-Madden**
Misspellings, double words

### CODE REVIEW

**Comment by Jak-MS** [comment]
@TylerMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/mfc/reference/cmdichildwndex-class.md [hunk 1]
```diff
@@ -953,7 +953,7 @@ BOOL IsTaskbarTabsSupportEnabled();
 
 ### Return Value
 
-TRUE if the MDI child can appear on Windows 7 taskbar tabs; FALSE if the MDI child can not appear on Windows 7 taskbar tabs.
+TRUE if the MDI child can appear on Windows 7 taskbar tabs; FALSE if the MDI child cannot appear on Windows 7 taskbar tabs.
```
Comment (TylerMSFT) [inline docs/mfc/reference/cmdichildwndex-class.md:957]: We prefer can't to cannot. No big deal. Just FYI for the future.

Comment (Blake-Madden) [reply docs/mfc/reference/cmdichildwndex-class.md:957]: Gotcha

**Review by TylerMSFT — APPROVED** [review]
Thank you, @Blake-Madden

**Comment by TylerMSFT** [comment]
#sign-off

**Pull request merged**